query-engine/core/src/schema_builder/input_types/objects/order_by_objects.rs

```diff
@@ -45,3 +52,31 @@ pub(crate) fn order_by_object_type(
     input_object.set_fields(fields);
     Arc::downgrade(&input_object)
 }
+
+fn order_by_object_type_aggregate(
+    ctx: &mut BuilderContext,
+    model: &ModelRef,
+    ordering_enum: &Arc<EnumType>,
+) -> InputObjectTypeWeakRef {
+    let ident = Identifier::new(format!("{}OrderByAggregateInput", model.name), PRISMA_NAMESPACE);
```

Comment:
Subjective: Not a fan of one-line-one-newline-one-line style :P

Reply:
Yeaaah, me neither. It's just that I like to separate var declarations from business logic/mutations, and it happens that on this function, we have: one var dec, one mutation, one var dec, one mutation, ... 😬. I'll move them like:

```
var dec
mutation

var dec
mutation

var dec
mutation

return
```